packages/shared/components/DelegationActivityDetails.svelte

```diff
@@ -0,0 +1,18 @@
+<script lang="ts">
+    import { AddressBox, ActivityInclusionStatusPill, AmountBox } from '@ui'
+    import { DelegationActivity, getAssetFromPersistedAssets } from '@core/wallet'
+    import { getCoinType } from '@core/profile'
+
+    export let activity: DelegationActivity
+
+    const asset = getAssetFromPersistedAssets(getCoinType())
+    $: amount = Number(activity?.delegatedAmount)
```

Comment:
What's the point of storing the `delegatedAmount` as a string if we are going to convert it into a Number later on. We could just convert it into a number when the activity is generated.